discordrb.gemspec

```
@@ -25,13 +25,13 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'ffi', '>= 1.9.24'
   spec.add_dependency 'opus-ruby'
-  spec.add_dependency 'rbnacl', '~> 5.0'
+  spec.add_dependency 'rbnacl', '>= 5.0'
```

I know you said this is relaxed so we can use 6.0, but I'd like to avoid `>=` if we can, and prefer `~>` whenever possible. Is there a particular reason we can't just `~> 6.0`? I glanced at the 6.0 release and I don't see why not.
```suggestion
  spec.add_dependency 'rbnacl', '~> 6.0'
```

---

Agreed. In maintaining discordrb, my line of thinking is if it has a ver restriction, I'm going to leave it like that because I assume it's there for a reason (do we rely on it?), or I don't otherwise know about how the deps API has changed over time. Also with something like our voice API, I'd lean towards keeping everything consistent..

`rest-client ~> 2.0` comes to mind was one that we added actually, because there was an *extremely* obscure error that people would come in and report because they had *ancient* versions of `rest-client` installed somehow that had a bunch of missing constants, completely breaking the lib. (there were at least 3 cases of this..)

Anyways, I like the 6.0 release - let's do that.

Also: I see now why you updated the wiki. :+1: